### DISCUSSION TITLE
How do you view a series of books in Plex?

### DISCUSSION THREAD

**Discussion by hvddrift** [seanap/Plex-Audiobook-Guide#24] [Q&A]
I have some audiobooks that have the series tag properly allocated. What is the best way to view a series in Plex?

**Comment by seanap** [comment]
Plex doesn't officially support Audiobooks, so the Plex Music library type doesn't have a robust series or collection view.  You can manually create a collection tag for the series, but there is currently no automated way to import the series tag to plex collection.

There are apps such as Prologue and BookCamp that will read the series tags imbedded in the file. You should be using these dedicated audiobook apps to play/browse the audiobooks, and not through plex itself.

**Reply by hvddrift** [reply]
Thank you. I am using an Android so Prologue is out, so I will have to wait for Bookcamp to allow me to join.

Thanks
